services/web/src/components/Auth/Google/SignInButton.js

```diff
@@ -7,7 +7,7 @@ import { signInWithGoogle } from 'utils/auth/google';
 
 import logo from 'assets/google-logo.svg';
 
-import './google.less';
+//import './google.less';
```

Comment:
do we not need the CSS? or can we remove this?

Reply:
I need to come back to this, i didnt test that part at all.